vscode-extension/editor/src/VSCodeEditor.tsx

```diff
@@ -233,11 +233,20 @@ export default function VSCodeEditor() {
 
   const deleteModelSettings = useCallback(
     async (modelName: string) => {
-      return await ufetch.post(ROUTE_TABLE.DELETE_MODEL(aiConfigServerUrl), {
-        model_name: modelName,
-      });
+      const res = await ufetch.post(
+        ROUTE_TABLE.DELETE_MODEL(aiConfigServerUrl),
+        {
+          model_name: modelName,
+        }
+      );
+
+      if (vscode) {
+        notifyDocumentDirty(vscode);
+      }
```

Comment:
are all the api calls notifying document dirty?

Reply:
Anything that modifies the aiconfig state (so, excludes api calls like list_models)